### ISSUE TITLE
[Feature] Red '...' when completing like in zsh

### ISSUE THREAD

**Issue by pickfire** [fish-shell/fish-shell#2131]

**Comment by ridiculousfish** [comment]
Can you please elaborate?

**Comment by lledey** [comment]
It may be a duplicate of #2109 ?

**Comment by pickfire** [comment]
No, it is not a duplicate of #2109. Taking a screenshot would be hard as the red dots only appear when waiting for tab completion. Look at https://github.com/robbyrussell/oh-my-zsh/blob/master/templates/zshrc.zsh-template#L28-L29

**Comment by ridiculousfish** [comment]
So it's like a progress indicator? Slick.

**Comment by pickfire** [comment]
Slick?

**Comment by pickfire** [comment]
No, it is just a completing indicator, it is not showing any progress.

**Comment by ridiculousfish** [comment]
I mean I think it's a nice feature.

**Comment by pickfire** [comment]
Yeah, but not seen by someone who have a super computer (almost no loading time).

**Comment by derekstavis** [comment]
:+1: for this! visual feedback for long-running completions is a _must_ :dancer:
